endoreg_db/serializers/sync/label_video_segment.py

```
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+from endoreg_db.models import LabelVideoSegment
+# from .video import VideoSerializer
+from .patient_finding import PatientFindingSerializer
```

Ruff (0.8.2)

`.patient_finding.PatientFindingSerializer` imported but unused

Remove unused import: `.patient_finding.PatientFindingSerializer`

(F401)